doc/man/salt.7

```diff
@@ -445612,7 +445612,7 @@ lists of expected minions before giving up. This value defaults to \fB5\fP secon
 The \fBsyndic_wait\fP setting is necessary because the higher\-level master does not
 have a way of knowing which minions are below the syndics. The higher\-level master
 has its own list of expected minions and the masters below them have their own lists
-as well, so the Salt client does not how long to wait for all returns. The
+as well, so the Salt client does not know how long to wait for all returns. The
```

Comment:
Almost missed this... we are not supposed to directly edit the man pages. These are generated at release time.